components/tls/tls-mpc/src/leader.rs

```diff
@@ -102,18 +108,22 @@ impl MpcTlsLeader {
         }
     }
 
+    /// Sets the protocol version
     pub fn set_protocol_version(&mut self, version: ProtocolVersion) {
         self.conn_state.protocol_version = Some(version);
     }
 
+    /// Sets the cipher suite
     pub fn set_cipher_suite(&mut self, suite: CipherSuite) {
         self.conn_state.cipher_suite = Some(suite);
     }
 
+    /// Sets the server random
     pub fn set_server_random(&mut self, random: Random) {
         self.conn_state.server_random = Some(random);
     }
 
+    /// Sets the server key
```

Comment:
/// Sets the server ephemeral key
and also pls change server_public_key()' comment below to be
/// Returns the server's ephemeral public key.